newrelic-alerts-configurator/README.md

```diff
@@ -305,6 +316,38 @@ ExternalServiceCondition apmExternalServiceCondition = ApmExternalServiceConditi
     .build();
 ```
 
+### Alerts NRQL condition
+
+NRQL Condition creation/update/removal policy is described in [Condition management policy](#condition-management-policy)
+
+To create NRQL condition for your alerts policy use simple builder:
+
+```java
+NrqlCondition.builder()
+```
+
+What you can set for NRQL condition:
+- condition name - Name of your NRQL condition.
+- enabled (optional) - If your NRQL condition is enabled. Default is false.
```

Comment:
Right now every type of condition works this way. This would be another breaking change. But I agree It would be more intuitive to have it enabled by default. I don't know why it is disabled as default in the first place. Maybe someone else would know?